src/index.html

```diff
@@ -10,13 +10,101 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Moyo header</title>
+    <title>Moyo Header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="https://www.moyo.ua/ua/">
+        <img
+          src="images/logo.png"
+          alt="MOYO Logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="list__item">
+            <a
+              class="is-active nav__link"
+              href="#"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="list__item">
```

Comment:
Add the blank line between the links 
```suggestion
          </li>
          
          <li class="list__item">
```